publish-package/README.md

```diff
@@ -0,0 +1,51 @@
+# Publish-PyPI-Package
```

Comment:
I think publish-package is a bit too broad of a name, I would suggest renaming the task to something like : 

python/publish-package.yaml

and bonus point we could move every python tasks in that repository for the user to make it easy to find...